python/ray/rllib/models/multiagentfcnet.py

```diff
@@ -37,10 +37,22 @@ def _init(self, inputs, num_outputs, options):
 
         # check for a shared model
         shared_model = custom_options.get("multiagent_shared_model", 0)
+        shared_model = shared_model or isinstance(shared_model, list) # for multiple shared models
+        shared_model_list = custom_options.get("multiagent_shared_model", 0) # how many models should be shared
```

Comment:
This is a bit confusing. The variable name implies that this is a list, but the default value is an integer and the comment also suggests that this is an integer.